platforms/allay/src/main/java/com/dfsek/terra/allay/Mapping.java

```diff
@@ -85,12 +84,15 @@ private static void error() {
     }
 
     private static boolean initBiomeMapping() {
-        try (InputStream stream = Mapping.class.getClassLoader().getResourceAsStream("mapping/biomes.json")) {
-            if  (stream == null) {
+        try(InputStream stream = Mapping.class.getClassLoader().getResourceAsStream("mapping/biomes.json")) {
```

Comment:
Why the blank after `try` keyword being removed? Looks like an error reformatting (

Reply:
No, this is the formatting the project currently uses (albeit I would prefer if not, but that's a separate discussion to be had).
See: the `.editorconfig`

Reply:
Got it